ui/pages/first-time-flow/seed-phrase/reveal-seed-phrase/reveal-seed-phrase.component.js

```diff
@@ -76,6 +79,15 @@ export default class RevealSeedPhrase extends PureComponent {
       },
     });
 
+    this.context.trackEvent({
+      category: EVENT.CATEGORIES.ONBOARDING,
+      event: EVENT_NAMES.NEW_WALLET_CREATED,
+      properties: {
+        action: 'Onboarding Complete',
+        legacy_event: true,
+      },
+    });
+
     await Promise.all([setCompletedOnboarding(), setSeedPhraseBackedUp(false)]);
```

Comment:
I don't think this is a blocker, but we could move the new `EVENT_NAMES.NEW_WALLET_CREATED`, action: `Onboarding Complete`, underneath this await to better ensure that the onboarding is complete